src/config.ts

```diff
@@ -1,7 +1,7 @@
+import { QueueInterface } from "core/queue/queue";
 import { ethers } from "ethers";
 
 import MemoryQueue from "./core/queue/memoryQueue";
-import { QueueInterface } from "./core/queue";
```

Comment:
why did this change? i kind of prefer having the external imports separated from the internal ones.

Reply:
I have no idea. My editor was complaining and I just had it do its thang. Let me take a look.

Reply:
It looks like my editor needed to be refresh. thanks for catching this.